webrender/src/resource_cache.rs

```diff
@@ -546,13 +573,14 @@ impl ResourceCache {
         let glyph_key_cache = self.cached_glyphs.get_glyph_key_cache_for_font(&font);
 
         for (loop_index, key) in glyph_keys.iter().enumerate() {
-            let glyph = glyph_key_cache.get(key);
-            let cache_item = glyph.as_ref().map(|info| self.texture_cache.get(&info.texture_cache_handle));
-            if let Some(cache_item) = cache_item {
-                f(loop_index, &cache_item.uv_rect_handle);
-                debug_assert!(texture_id == None ||
-                              texture_id == Some(cache_item.texture_id));
-                texture_id = Some(cache_item.texture_id);
+            if let Some(glyph) = glyph_key_cache.get(key) {
```

Comment:
We need to handle the case of failure here. Does it make sense to return as usual if some of the glyphs failed to be obtained?